emails/utils.py

```diff
@@ -41,10 +44,55 @@
 study_logger = logging.getLogger("studymetrics")
 metrics = markus.get_metrics("fx-private-relay")
 
-with open("emails/tracker_lists/level-one-tracker.json", "r") as f:
-    GENERAL_TRACKERS = json.load(f)
-with open("emails/tracker_lists/level-two-tracker.json", "r") as f:
-    STRICT_TRACKERS = json.load(f)
+shavar_prod_lists_url = "https://raw.githubusercontent.com/mozilla-services/shavar-prod-lists/master/disconnect-blacklist.json"
+EMAILS_FOLDER_PATH = pathlib.Path(__file__).parent
+TRACKER_FOLDER_PATH = EMAILS_FOLDER_PATH / "tracker_lists"
+
+
+def get_trackers(level):
```

Comment:
I am thinking this can be tested maybe on Circle CI as an additional step under build docker (e.g. open the file and check that there is content) or have the `FileNotFound` error be raised if the exception happens outside of dev.